user_guide/common_features.rst

```diff
@@ -0,0 +1,79 @@
+.. _user-guide-common-features:
+
+Common Features
+===============
+
+Customers, Channels and Accounts
+--------------------------------
+
+One of the main tasks for any CRM is ability to aggregate the details of a customer. 
+For example, you have two web-based stores, a retail outlet and a factory. There is a company whose managers 
+purchase from your web-based stores and with whom you have some business arrangements at the 
+factory. 
+
+Of course, there are much more details and peculiarities, but if we keep it simple, in an on-line store, somebody comes 
+to the store's website, looks through the items offered, and if interested in 
+buying them puts the items in the cart and orders them, leaving you the contact and shipping details. In many cases the 
+web-store customers can log-in and their profile details will be used. What kind of information do you need to make 
+relations with such customers successful? The most basic things are - what pages have people viewed, what items they 
+have ordered, if there is anything that a customer has added to the cart but hasn't ordered and if there are some 
+complaints after the delivery.
```

Comment:
Could you clarify this for me? Dima told me to review all sections in green
from this pull request:
https://github.com/orocrm/documentation/pull/198/files.

I'm new to GitHub though, so maybe I misunderstood something? Have I
completed my work for this section, or should I continue?

David

On Wed, Mar 9, 2016 at 2:12 AM, nnenasheva <notifications@github.com> wrote:

> In user_guide/common_features.rst
> <https://github.com/orocrm/documentation/pull/198#discussion_r55496779>:
>
> > +
> > +Customers, Channels and Accounts
> > +--------------------------------
> > +
> > +One of the main tasks for any CRM is ability to aggregate the details of a customer.
> > +For example, you have two web-based stores, a retail outlet and a factory. There is a company whose managers
> > +purchase from your web-based stores and with whom you have some business arrangements at the
> > +factory.
> > +
> > +Of course, there are much more details and peculiarities, but if we keep it simple, in an on-line store, somebody comes
> > +to the store's website, looks through the items offered, and if interested in
> > +buying them puts the items in the cart and orders them, leaving you the contact and shipping details. In many cases the
> > +web-store customers can log-in and their profile details will be used. What kind of information do you need to make
> > +relations with such customers successful? The most basic things are - what pages have people viewed, what items they
> > +have ordered, if there is anything that a customer has added to the cart but hasn't ordered and if there are some
> > +complaints after the delivery.
>
> Thanks so much for your comments and edits, they are really valuable.
> Please note, though, that this pull request is only about the Getting
> Started and Data Management sections. Business intelligence and common
> features are a part of the next PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/orocrm/documentation/pull/198/files#r55496779>.
>